app/sagas/login.js

```
@@ -33,7 +33,7 @@ import { getUserPresence, subscribeUsersPresence } from '../lib/methods/getUsers
 import { logout, removeServerData, removeServerDatabase } from '../lib/methods/logout';
 import { subscribeSettings } from '../lib/methods/getSettings';
 import { disconnect, loginWithPassword, login } from '../lib/services/connect';
-import { saveUserProfile, registerPushToken, getUsersRoles, setUserPresenceAway } from '../lib/services/restApi';
+import { saveUserProfile, registerPushToken, getUsersRoles, setUserPresenceAway, pendingToken, pendingVoipToken } from '../lib/services/restApi';
```

ESLint

[error] Replace `·saveUserProfile,·registerPushToken,·getUsersRoles,·setUserPresenceAway,·pendingToken,·pendingVoipToken·` with `⏎↹saveUserProfile,⏎↹registerPushToken,⏎↹getUsersRoles,⏎↹setUserPresenceAway,⏎↹pendingToken,⏎↹pendingVoipToken⏎`

(prettier/prettier)